README.md

```diff
@@ -33,8 +32,7 @@ output "resource_group_id" {
 New resource group
 ```hcl
 module "resource_group" {
-  # Replace "main" with a GIT release version to lock into a specific release
-  source                       = "git::https://github.com/terraform-ibm-resource-group/terraform-ibm-resource-group.git?ref=main"
+  source  = "../.."
```

Comment:
This is the wrong format for a README. It should be source and version = latest with the comment updated.